src/plugin.js

```diff
@@ -772,56 +365,21 @@ const contribAdsPlugin = function(options) {
     player.textTracks().addEventListener('change', textTrackChangeHandler);
   });
 
-  // Register our handler for the events that the state machine will process
-  player.on(VIDEO_EVENTS.concat([
-    // Events emitted by this plugin
-    'adtimeout',
-    'contentupdate',
-    'contentplaying',
-    'contentended',
-    'contentresumed',
-    // Triggered by startLinearAdMode()
-    'adstart',
-    // Triggered by endLinearAdMode()
-    'adend',
-    // Triggered by skipLinearAdMode()
-    'adskip',
-
-    // Events emitted by integrations
-    'adsready',
-    'adserror',
-    'adscanceled',
-    'nopreroll'
-
-  ]), processEvent);
+  // Event handling for the current state.
+  player.on([
```

Comment:
I believe `VIDEO_EVENTS` is still defined in this file, is it unused now? Also will this change cause html5 events that are currently getting redispatched now to not be handled? If so, is that a change you were intending to make?

Reply:
VIDEO_EVENTS is used for redispatch. This change does not impact redispatch.